TRENZ.Lib.RazorMail.MailKit/MailKitMailSender.cs

```diff
@@ -65,12 +64,9 @@ public override async Task SendAsync(SmtpAccount account)
         foreach (var item in ReplyTo)
             mail.ReplyTo.Add(item.ToMailboxAddress());
 
-        Log.Info(
-            $"Sending mail from {mail.From} to {string.Join(", ", mail.To)}, cc {string.Join(", ", mail.Cc)}, bcc {string.Join(", ", mail.Bcc)}");
-
-        Log.Info($"{nameof(mail.Subject)}: {mail.Subject}");
-
-        Log.Debug(mail.Body);
+        Logger.LogInformation("Sending mail from {From} to {Recipients} (CC: {Cc}, BCC: {Bcc}", mail.From, mail.To, mail.Cc, mail.Bcc);
```

Comment:
does this actually expand the collections?

Reply:
![image](https://github.com/trenz-gmbh/TRENZ.Lib.RazorMail/assets/6266356/3e874063-a2c8-4d62-80e7-082abd6a7052)

It does, but it looks like I forgot a parenthesis...